contracts/Tokens/VTokens/VToken.sol

```diff
@@ -302,6 +323,29 @@ contract VToken is VTokenInterface, Exponential, TokenErrorReporter {
         return getCashPrior();
     }
 
+    /**
+     * @notice Governance function to set new threshold of block difference after which funds will be sent to the protocol share reserve
+     * @param newReduceReservesBlockDelta_ block difference value
+     */
+    function setReduceReservesBlockDelta(uint256 newReduceReservesBlockDelta_) external returns (uint) {
+        require(newReduceReservesBlockDelta_ > 0, "Invalid Input");
+        ensureAllowed("setReduceReservesBlockDelta(uint256)");
+        emit NewReduceReservesBlockDelta(reduceReservesBlockDelta, newReduceReservesBlockDelta_);
+        reduceReservesBlockDelta = newReduceReservesBlockDelta_;
+    }
+
+    /**
+     * @notice An admin function to set new threshold of block difference after which funds will be sent to the protocol share reserve
```

Comment:
This comment is wrong, copied from `setReduceReservesBlockDelta`

Reply:
Fixed here [5312427](https://github.com/VenusProtocol/venus-protocol/pull/262/commits/531242717019d18408ac659f14e3fe622206371b)